zgw/documenten-api/src/main/kotlin/com/ritense/documentenapi/client/DocumentenApiClient.kt

```diff
@@ -206,33 +193,27 @@ class DocumentenApiClient(
             }
             .accept(MediaType.APPLICATION_OCTET_STREAM)
             .retrieve()
-            .bodyToFlux<DataBuffer>()
-            .toInputStream {
-                TransactionTemplate(platformTransactionManager).executeWithoutResult {
-                    outboxService.send {
-                        DocumentInformatieObjectDownloaded(
-                            objectUrl.toString()
-                        )
-                    }
-                }
+            .body<String>()!!
```

Comment:
By making it a `String` the entire document is loaded in-memory. 
Is it possible to change this to an `InputStream`?